src/test/java/HomeWork23.java

```diff
@@ -0,0 +1,26 @@
+import org.openqa.selenium.support.PageFactory;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+import pageObjects.ArtistsPage;
+import pageObjects.LoginPage;
+
+public class HomeWork23 extends BaseTest {
+    @Test
+
+    public void PageFactory() throws InterruptedException {
```

Comment:
Please, delete `throws InterruptedException`

Thread. sleep is bad! It blocks the current thread and renders it unusable for further work.
Using Thread. sleep() frequently in an automation framework is not a good practice.

Please use the Explicit Wait command, the WebDriver is directed to wait until a certain condition occurs before proceeding with executing the code.